src/extras/DataUtils.js

```diff
@@ -8,144 +8,154 @@ class DataUtils {
 
 	static toHalfFloat( val ) {
 
+		if ( ! this._floatView ) this._generateTables();
```

Comment:
Better to use a module scope variable here and not polluting the public namespace.